style.css

```diff
@@ -0,0 +1,280 @@
+	background: #fafafa;
+}
+.socialmedia h3{
+	color: #555;
+	font-size: 2.5rem;
+	font-weight: bold;
+	padding-bottom: 0.63rem;
+}
+.fab{
+  padding: 0.63rem;
+  font-size: 1.25rem;
+  text-decoration: none;
+  margin: 0.31rem 0.13rem;
+  border-radius: 50%;
+  margin-right: 0.63rem;
+}
+.fab:hover{
+  opacity: 0.6;
+}
+.fa-instagram{
+   background: #e4405f;
+   color: white;
+}
+.fa-twitter {
+  background: #55ACEE;
+  color: white;
+}
+.fa-linkedin-in {
+  background: #007bb5;
+  color: white;
+}
+.fa-github{
+  background: purple;
+  color: white;
+}
+.fa-quora{
+  background: #b92b27;
+  color: white;
+}
+.about{
+	padding-top: 0.63rem;
+	padding-bottom: 3.13rem;
+}
+.about h1{
+	text-align: center;
+	color: #ea213b;
+	font-family: 'Times New Roman';
+}
+.about h1::after{
+	content: '';
+	display: block;
+	background: #ea213b;
+	height: 2px;
+	width: 100px;
+	margin: 20px auto 5px;
+	margin-bottom: 50px;
+}
+.about .container{
+	margin: 3.13rem auto;
+}
+.about-col{
+	margin-top: 3.13rem;
+}
+.profile{
+	padding: 4.38rem 0.63rem 1.25rem 0.63rem;
+	background-color: #ea213b;
+}
+.profile:hover{
+	box-shadow: 0 0 15px 5px rgba(0,0,0,0.4);
+	cursor: pointer;
+	transition: 0.5s;
+}
+.profile > p{
+	color: #fff;
+	font-size: 1rem;
+	padding-top: 1.25rem;
+	padding-bottom: 1.25rem;
+}
+.profile img{
+	width: 7.5rem;
+	height: 7.5rem;
+	border-radius: 50%;
+	top: -3.75rem;
+	position: absolute;
+	left: calc(50% - 60px);
+	border: 7px solid #fff;
+}
+.skills-bar > p{
+	margin-bottom: 1.25rem;
+	font-weight: 600;
+	font-size: 1.13rem;
+}
+.progress{
+	border-radius: 0.94rem !important;
+	margin-bottom: 0.94rem;
+	margin-top: 0.19rem;
+}
+.progress-bar{
+	border-radius: 0.94rem;
+	background-color: #ea213b !important;
+}
+.contact-form{
+	background-color: #fafafa;
+	padding: 3.13rem;
+	margin-bottom: 1.88rem;
+}
+.form-group{
+	width: 60%;
+}
+.form-control{
+	outline: none;
+}
+.contact-form h2{
+    font-family: 'Pacifico', cursive !important;
+}
+.contact-form button{
+	padding: 0.63rem 1.88rem;
+}
+.footer{
+	background-image: linear-gradient(rgba(0,0,0,0.7), rgba(0,0,0,0.7)), url(https://i.pinimg.com/originals/e1/e8/8e/e1e88e69109c22068694d3894e649370.jpg);
+	/* background-color: #2d314d; */
+	color: #fff;
+	padding-top: 0.63rem;
+}
+.lg{
+	padding-left: 3.13rem !important;
+	padding-top: 1.25rem !important;
+}
+.lg h5{
+    font-family: 'Pacifico', cursive;
+    font-size: 1.88rem;
+}
+.lg span{
+	color: #ea213b;
+}
+.fas, .far{
+	font-size: 1.25rem;
+	margin-top: 1.25rem;
+	opacity: 0.5;
+}
+.sos-med{
+	margin-left: -9.375rem !important;
+	padding-top: 3.13rem !important;
+}
+.list-unstyled{
+    margin-left: 2.5rem;
+}
+.list-unstyled span{
+	font-family: georgia;
+	font-size: 0.88rem;
+	color: #efefef;
+}
+@media screen and (max-width: 768px) {
+    .homepage {
+      height: 110vh; 
+    }
+	.home-details p{
+		font-size: 40px;
+	}
+    .form-group {
+      width: 100%;
+    }
+  
+    .sos-med{
+    margin-left: 30px !important;
```

Comment:
Please fix your indentation. Also use `rem` and try to make this stick without using !important.